demisto_sdk/commands/common/tools.py

```diff
@@ -2004,3 +2004,7 @@ def suppress_stdout():
             yield
         finally:
             sys.stdout = old_stdout
+
+def is_iron_bank_pack(file_path):
+    metadata = get_pack_metadata(file_path)
+    return 'Iron Bank' in metadata.get('tags', [])
```

Comment:
Don't we have a constants file for a pack's tags?

Reply:
No, but I'll add a constant instead.